### PR TITLE
SB-18325(print-service apis added)

### PR DESCRIPTION
**Pull request by anmol2302**
**Ticket Ref:** [SB-18325](https://project-sunbird.atlassian.net/browse/SB-18325)
**Target Release:** 2.9.0
APIs Added
 - /v1/print/pdf
 - /v1/print/preview/generate

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://573.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://573.qa.docs.sunbird.org/2.7.0/apis/printservice// <br /> - http://573.qa.docs.sunbird.org/2.7.0/apis/printservice// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @anmol2302,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://573.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://573.qa.docs.sunbird.org/2.7.0/apis/printservice// <br /> - http://573.qa.docs.sunbird.org/2.7.0/apis/printservice// <br />  <br/> **Modified sections are -** <br />

File: apis/printservice/printservice.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                type: string
+            context:
+                description: This will contain the placeHolder and it's value which needs to be replaced in HTML.
+                type: object
+                example:
+                    recipientName: John Deo
+                    certificateName: ABC Certificate
+            storageParams:
+                type: object
+                description: this object will have the container name and path where its needs to be stored in cloud. it not provided it will stored at default location.
```
Comment (Sunitashelar) [inline apis/printservice/printservice.yaml:117]: @anmol2302 modify the sentence to, 'This object has the container name and path where the certificate needs to be stored. If this is not provided, the system stores the certificate in the default location.

File: apis/printservice/printservice.yaml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    PrintPdfResponse:
+        description: PDF response
+        type: object
+        properties:
+            result:
+                description: The response details
+                type: object
+                properties:
+                    pdfUrl:
+                        description: The downloadable url of the pdf, public in nature
```
Comment (Sunitashelar) [inline apis/printservice/printservice.yaml:142]: @anmol2302 modify the sentence to, 'The public URL from where the pdf can be downloaded.'

File: apis/printservice/printservice.yaml [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+                        type: string
+                    ttl:
+                        description: The time to live of URl, after ttl time pdf link will get expired.
```
Comment (Sunitashelar) [inline apis/printservice/printservice.yaml:145]: @anmol2302 please clarify.

**Comment by NupurSancheti** [comment]
@Hari-stackroute: As discussed, please resolve the queries.

**Comment by Sunitashelar** [comment]
Closing the PR as the changes requested for were not done.

**Pull request closed without merging**